src/docs/product/issues/index.mdx

```diff
@@ -29,7 +31,21 @@ To see the stack trace of the latest event, you can hover over the issue title.
 
 ![Error level icon](error-level-icon.png)
 
-When you click on an issue on the main **Issues** page, the [Issue Details](/product/issues/issue-details) page displays.
+<!-- update image for performance issues GA -->
```

Comment:
@gggritso - replace image and remove or just remove if image doesn't need updating.

Reply:
This one can be left as-is, removing the comment!